### ISSUE TITLE
Class Bar resets to (0,0) when trying to move

### ISSUE THREAD

**Issue by nathanielbelen** [tullamods/Dominos#869]
**Addon Version**
You can get this via running `/dominos version` in game
10.2.36-cata

**World of Warcraft Version**
Classic Cataclysm

On a fresh install + no addons but dominos + fresh reset profile, whenever I try to move the class bar on my Survival hunter, it keeps resetting to 0,0.

https://streamable.com/mzx0ia

**Comment by Tuller** [comment]
I've not replicated this, at least on the PTR with a fresh hunter.

**Comment by causticsudo** [comment]
same problem, here.
but, just happened on dk, or paladin without troubles

**Comment by Beagleholic** [comment]
same, too.
it happens when only hunter in my case.

**Comment by Tuller** [comment]
Is this still ocurring?

**Comment by nathanielbelen** [comment]
It is but it's hard to replicate it. Some times when I try to move it, it works, and others it's impossible to move from 0,0. I'm pretty sure I'm not changing anything settings-wise either, really feels like a random bug

**Comment by Wohngebietsuwe** [comment]
> Is this still ocurring?

Yes, it is. For some time it helped to reset to Dominos v10.2.33, but this is over since yesterday. You CAN move the class bar with the keyboard's movement keys, though (but slooooooowly ^^). But with my mouse it's the same behaviour like shown in the video.

**Comment by SladeThe** [comment]
I can confirm. My Death Knight's class bar is stuck in the left bottom corner.

**Comment by HuMa69** [comment]
UP  its still bugged, it works on my rogue but doesnt work on DK, it reset to a corner

**Comment by darrenwri** [comment]
This happened to me yesterday, 22 June 2025.  Only one character, my hunter, has this issue.  I was able to replicate the problem with only Dominos running, no other addons.
